CONTRIBUTING.md

```diff
@@ -85,6 +86,48 @@ A lot of the instructions below replicate what the [PR CI pipeline] is doing.
 1. You are now ready to submit your PR.
 1. After your PR ise merged, most likely you will want to read [`How to deploy your changes`](#how-to-deploy-your-changes) to verify they got deployed.
 
+# (WIP) New linter rule promotion strategy
```

Comment:
All newly added rules should have severity of `off`, to ensure they run only in staging, not prod. For an example, see:

- https://github.com/Azure/azure-openapi-validator/pull/532

Reply:
@bdefoy actually, we decided to adopt different approach. The severity should remain as intended, but the rules should have additional property: `stagingOnly: true`

Reply:
Also, with this new staging setup, we probably should set rule severity to `off` only if it is totally busted and we don't want for it to run at all, not even in staging.